src/components/containers/notification/Notification.js

```diff
@@ -152,6 +152,7 @@ function renderCallsToAction(callsToAction) {
           <Button
             styleType={buttonStyleType}
             key={index}
+            theme={theme}
```

Comment:
Related to #123, where button has it's own `ThemeProvider` that defaults to using the default theme when we don't explicitly pass in the theme like this.